### PR TITLE
Ankush/mvp 4086

### PR DESCRIPTION
**Pull request by ankush-notifi**
- [ ] Describe the purpose of the change

- [ ] Create test cases for your changes if needed

- [ ] Include the ticket id if possible (Collaborator only)

https://notifi.atlassian.net/browse/MVP-4086

### CODE REVIEW

File: packages/notifi-dapp-example/src/components/Toggle.tsx [hunk 1]
```diff
@@ -4,12 +4,14 @@ export type ToggleProps = Readonly<{
...
   checked: boolean;
   onChange?: () => void;
+  handleChange?: (val: boolean) => void;
 }>;
 
 export const Toggle: React.FC<ToggleProps> = ({
   disabled,
   checked,
   onChange,
+  handleChange,
```
Comment (eric-notifi) [inline packages/notifi-dapp-example/src/components/Toggle.tsx:14]: Do we need an additional handle change? 
Looks like we can simply utilize `onChange`

**Review by eric-notifi — CHANGES_REQUESTED** [review]
Generally looks nice. Some small nits herewith.
It seems going to have some conflicts with [this PR](https://github.com/notifi-network/notifi-sdk-ts/pull/482). There are some changes according to the repo convention. Suggest to take a look at that PR and Rebase that one.
And looks like the real test in my local env (dev tenant) is not working well. Do you run into the same issue (check the video herewith).

https://github.com/notifi-network/notifi-sdk-ts/assets/127958634/f5c92cf3-54f6-484d-bef8-3017ec4f39e0

And the github action seems not going through. might need to take a look. package-lock.json issue?

@ankush-notifi @marukohao

File: packages/notifi-dapp-example/src/components/Toggle.tsx [hunk 2]
```diff
@@ -23,7 +23,9 @@ export const Toggle: React.FC<ToggleProps> = ({
         className="sr-only peer"
         checked={checked}
         disabled={disabled}
-        onChange={onChange}
+        onChange={() => {
+          onChange?.();
```
Comment (marukohao) [inline packages/notifi-dapp-example/src/components/Toggle.tsx:27]: Why do we need to use optional chaining here?

**Comment by marukohao** [comment]
Slack works good for sign up, when edit destinations, toggle on then toggle off the slack, get an error message 'Failed to save", and when check on the networks, seems we didn't call the updateTarget query.
Screenshot 2024-02-29 at 1 46 51 PM

Untitled

File: packages/notifi-dapp-example/src/components/Toggle.tsx (hunk 2 above)
Comment (ankush-notifi) [reply packages/notifi-dapp-example/src/components/Toggle.tsx:27]: since `onChange` is optional in props, so It's safe to use optional chaining here.

**Comment by marukohao** [comment]
LGTM, can you fix the action fail before merging?

**Review by eric-notifi — APPROVED** [review]
LGTM

**Pull request merged**